<launch>

  <arg name="device_id" value="1"/>
  <arg name="device_num" value="2"/>
  <arg name="topic_send_way" value="false"/>

  <node pkg="swarm_bridge" name="bridge_node" type="bridge_tcp_node" output="screen">
  
    <param name="self_id" value="$(arg device_id)"/>
    <param name="is_ground_station" value="false"/>
    <param name="max_freq" value="1000"/>
    <param name="device_num" value="$(arg device_num)"/>
    <param name="topic_send_way" value="$(arg topic_send_way)"/>
    <param name="device_ip_0" value="192.168.50.218" type="string"/>
    <param name="device_ip_1" value="192.168.50.24" type="string"/>
    <param name="device_ip_2" value="127.0.0.1" type="string"/>
    <param name="device_ip_3" value="127.0.0.1" type="string"/>
    <param name="device_ip_4" value="127.0.0.1" type="string"/>
    <param name="device_ip_5" value="127.0.0.1" type="string"/>
    <param name="device_ip_6" value="127.0.0.1" type="string"/>
    <param name="device_ip_7" value="127.0.0.1" type="string"/>
    <param name="device_ip_8" value="127.0.0.1" type="string"/>
    <param name="device_ip_9" value="127.0.0.1" type="string"/>
    <param name="ground_station_num" value="0"/>
    <param name="ground_station_ip_0" value="127.0.0.1" type="string"/>
    <!-- <remap from="/imu_topic" to="/mobile_base/sensors/imu_data"/> -->
    <remap from="/cloud_xyz_topic" to="/cloud_xyz_topic"/>

  </node>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find swarm_bridge)/rviz_config/receive_map.rviz" required="true"/>
</launch>
